.github/workflows/security.yml

```diff
@@ -72,26 +72,29 @@ jobs:
     needs: trivy-repo
     strategy:
       matrix:
-        version:
-          - "latest"
+        include:
+          - channel: "3/edge"
+            branch: "main"
+          - channel: "2/stable"
```

Comment:
Maybe we can set this to `2/candidate` for now until `2/stable` is populated. We can then also uncomment `3/edge` in the same PR.